models/fates/R/write.configs.FATES.R

```diff
@@ -815,32 +384,16 @@ write.config.FATES <- function(defaults, trait.values, settings, run.id){
        # SAV	Surface Area to Volume Ratio of fuel class 	litterclass	cm-1
        
        ## NCWD dimensioned       Size:4
-       if(var == "CWD_frac1"){            ##Fraction of coarse woody debris (CWD) that is moved into each of the four woody fuel classes
-         ncdf4::ncvar_put(nc=fates.param.nc, varid='fates_CWD_frac', start = 1, count = 1,
-                   vals=pft[v])
-       }
-       if(var == "CWD_frac2"){
-         ncdf4::ncvar_put(nc=fates.param.nc, varid='fates_CWD_frac', start = 2, count = 1,
-                   vals=pft[v])
-       }
-       if(var == "CWD_frac3"){
-         ncdf4::ncvar_put(nc=fates.param.nc, varid='fates_CWD_frac', start = 3, count = 1,
-                   vals=pft[v])
-       }
-       if(var == "CWD_frac4"){
-         ncdf4::ncvar_put(nc=fates.param.nc, varid='fates_CWD_frac', start = 4, count = 1,
-                   vals=pft[v])
-       }
        
        
-     } ## end loop over VARIABLES
-   } ## end loop over PFTs
+     #} ## end loop over VARIABLES
+   #} ## end loop over PFTs
    #ncdf4::nc_close(param.nc)
-   ncdf4::nc_close(clm.param.nc)
-   ncdf4::nc_close(fates.param.nc)
+   #ncdf4::nc_close(clm.param.nc)
+   #ncdf4::nc_close(fates.param.nc)
```

Comment:
got to the end of this and do see where a new alternative way of parameter mapping ended up being implemented